index.html

```diff
@@ -19,29 +19,29 @@
         </nav>
     </header>
 
-        <section class="headline-section">
-            <h1 class="heading-1">I'm Abdul <br> Glad to see you!</h1>
-            <p class="intro-info">I’m a software developer! I can help you build a product , feature or website Look
-                through some of my work and experience! If you like what you see and have a project you need coded,
-                don’t hestiate to contact me.</p>
-            <h2 class="connect-info">LET’S CONNECT</h2>
-
-            <ul class="social-icons">
-                <li><a href="twitter.com"><img class="logo-images" src="./assets/icons/Twitter_logo.svg"
-                            alt="Twitter_logo"></a></li>
-                <li><a href="linkedin.com"><img class="logo-images" src="./assets/icons/Linkedin_icon.svg"
-                            alt="Linkedin Icon"></a></li>
-                <li><a href="medium.com"><img class="logo-images" src="./assets/icons/Medium.svg" alt="Medium logo"></a>
-                </li>
-                <li><a href="Github.com"><img class="logo-images" src="./assets/icons/Github_icon.svg"
-                            alt="Github logo"></a></li>
-                <li><a href="angelist.com"><img class="logo-images" src="./assets/icons/Angelist.svg"
-                            alt="Angelist logo"></a></li>
-            </ul>
-        </section>
+    <section class="headline-section">
+        <h1 class="heading-1">I'm Abdul <br> Glad to see you!</h1>
+        <p class="intro-info">I’m a software developer! I can help you build a product , feature or website Look
+            through some of my work and experience! If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.</p>
+        <h2 class="connect-info">LET’S CONNECT</h2>
+
+        <ul class="social-icons">
+            <li><a href="twitter.com"><img class="logo-images" src="./assets/icons/Twitter_logo.svg"
+                        alt="Twitter_logo"></a></li>
+            <li><a href="linkedin.com"><img class="logo-images" src="./assets/icons/Linkedin_icon.svg"
+                        alt="Linkedin Icon"></a></li>
+            <li><a href="medium.com"><img class="logo-images" src="./assets/icons/Medium.svg" alt="Medium logo"></a>
+            </li>
+            <li><a href="Github.com"><img class="logo-images" src="./assets/icons/Github_icon.svg"
+                        alt="Github logo"></a></li>
+            <li><a href="angelist.com"><img class="logo-images" src="./assets/icons/Angelist.svg"
+                        alt="Angelist logo"></a></li>
+        </ul>
+    </section>
     <!-- First Page -->
     <main class="work">
```

Comment:
- [ ] Kindly make sure your work section is wrapped in a `<section>` tag, just like the previous Code Reviewer required.